examples/java-matter-controller/java/src/com/matter/controller/commands/common/MatterCommand.java

```diff
@@ -41,6 +41,7 @@ public abstract class MatterCommand extends Command {
   private final AtomicLong mCommissionerNodeId = new AtomicLong();
   private final AtomicBoolean mUseMaxSizedCerts = new AtomicBoolean();;
   private final AtomicBoolean mOnlyAllowTrustedCdKeys = new AtomicBoolean();;
+  private String TestResult = null;
```

Comment:
Use name mTestResult, from kevin's previous comments, we should use  Optional<String>  in prevent compare with null for a none final object in our code.

Reply:
agree, done, thanks